scripts/workspace_deploy.py

```diff
@@ -93,13 +92,16 @@ def main(argv):
         logger.error('Cannot upload workspace.')
         sys.exit(1)
 
-    if not hasattr(config, 'conversation_workspace_id') or not getattr(config, 'conversation_workspace_id'):
+    if not getOptionalParameter(config, 'conversation_workspace_id'):
         setattr(config, 'conversation_workspace_id', responseJson['workspace_id'])
         logger.info('WCS WORKSPACE_ID: %s', responseJson['workspace_id'])
-    if hasattr(config, 'common_output_config'):
-        config.saveConfiguration(getattr(config, 'common_output_config'))
 
-    if hasattr(config, 'context_client_name'):
+    outputConfigFile = getOptionalParameter(config, 'common_output_config')
+    if outputConfigFile:
+        config.saveConfiguration()
```

Comment:
I think that we need to change this to:
`config.saveConfiguration(outputConfigFile)`

Reply:
Fixed, thank you!